### ISSUE TITLE
Variables cannot be duplicated if one already exists that is names copy_of itself

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46497]
**Steps to reproduce**:

using keyboard navigation, try to navigate template variable setup including variable duplication and renaming 

**Actual Result**:

Drag and drop UI issues: reordering does not stick (mouse or keyboard) when we don't rename the `copy_of_copy_of` variables. And variables cannot be duplicated if one already exists that is names `copy_of` itself

**Expected Result**

workflow accessible for keyboard-only users

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by TsotosA** [comment]
Hello, i believe #56300 should resolve this.
